<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginDlg</class>
 <widget class="QDialog" name="PluginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>353</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add or remove plugins</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../erebus.qrc">
    <normaloff>:/images/logo32.png</normaloff>:/images/logo32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listPlugins">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dlgButtonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../erebus.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>PluginDlg</receiver>
   <slot>onOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>PluginDlg</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>PluginDlg</receiver>
   <slot>onAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>clicked()</signal>
   <receiver>PluginDlg</receiver>
   <slot>onRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listPlugins</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>PluginDlg</receiver>
   <slot>onSelectionChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOk()</slot>
  <slot>onCancel()</slot>
  <slot>onAdd()</slot>
  <slot>onRemove()</slot>
  <slot>onSelectionChanged(int)</slot>
 </slots>
</ui>
